SECURITY.md

```diff
@@ -0,0 +1,3 @@
+# JavaFX Vulnerabilities
+
+Please follow the process outlined in the [OpenJDK Vulnerability Policy](https://openjdk.org/groups/vulnerability/report) to disclose vulnerabilities in JavaFX.
```

Comment:
since FX is not technically a part of JDK, should it point to a separate (new) page instead of https://openjdk.org/groups/vulnerability/report ?

Reply:
No. JavaFX _is_ part of OpenJDK. It is irrelevant whether or not it happens to be bundled with the JDK.

Reply:
+1